lib/galaxy/webapps/galaxy/services/tools.py

```diff
@@ -55,27 +64,40 @@ def __init__(
         self.toolbox_search = toolbox_search
         self.history_manager = history_manager
 
+    def create_temp_file(self, trans, files) -> Dict[str, FilesPayload]:
+        files_payload = {}
```

Comment:
Code suggestions are broken for me today...

But I suggest typing these parameters: `trans: ProvidesHistoryContext` and  `files: List[UploadFile]`